share/man/man9/rman.9

```diff
@@ -296,31 +289,20 @@ The alignment requirement
 .Pq if any
 is specified in
 .Fa flags .
-The
-.Fa bound
-argument may be set to specify a boundary restriction such that an
-allocated region may cross an address that is a multiple of the
-boundary.
-The
-.Fa bound
-argument must be a power of two.
-It may be set to zero to specify no boundary restriction.
+Often the
+.Dv RF_ALIGNMENT_LOG2
+macro is used to specify alignment to a power-of-2 size, or
+.Fn rman_make_alignment_flags
+can be used to compute the
+.Fa flags
+value at runtime.
```

Comment:
Note, `roff` isn't something I've worked with so this may or may not be right.  I'm kind of guessing, but this seemed to be right given what was here before.